conan/tools/microsoft/__init__.py

```diff
@@ -2,3 +2,4 @@
 from conan.tools.microsoft.msbuild import MSBuild
 from conan.tools.microsoft.msbuilddeps import MSBuildDeps
 from conan.tools.microsoft.visual import msvc_runtime_flag
+from conan.tools.microsoft.subsystems import unix_path
```

Comment:
Quick question, are the subsystems a ``microsoft`` thing, like ``msvc``, or a ``gnu`` thing?

Reply:
Hard to know, I would see it more "Microsoft" than "gnu". Take into account that not every "linux/unix" thing is "gnu", not even package config is really gnu.